spec/helpers/application_helper_spec.rb

```diff
@@ -25,4 +25,27 @@
       end
     end
   end
+
+  describe "title" do
+
+    context "nil input" do
```

Comment:
Can we move this to be the last test in the describe block please?. The happy path should be first ie test that it works with an input and then test the other branches and edge cases.